backend/LoadCell.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
+import matplotlib.pyplot as plt
+import json
+import numpy as np
+
+class CalLoadCell():
+    class State(Enum):
```

Comment:
For the naming of the States they are a little unclear.

NORMAL seems like it is when we want to begin calibrating and are waiting for our first real mass, which then puts us in the STORE state which is when we are waiting for the voltage that masses the previously received mass. Finally WAIT seems to be us just returning the weight based on the voltage after calibration.

Might be better to have them be something like EXPECTED_MASS, ACTUAL_VOLTAGE, and CALIBRATED.

@cjchanx  let me know what you think as well.

Reply:
Yeah the naming should be a bit more explicit, for example STORE_NEXT_VOLTAGE or CALIBRATION_IN_PROGRESS.

My original plan was to have the NORMAL/RUN state when the LoadCell class is basically doing nothing except for providing a slope/zero/tare to adjust LC values, but this can be adjusted. I think since the only time it'll get out of the NORMAL state is when it receives an expected_mass call it is ok to not have an explicit state for that (rather it is a state transition), but depends on how exactly the state machine is done.

![image](https://github.com/StudentOrganisationForAerospaceResearch/RocketControlUnit/assets/78698227/fd55f981-1946-4a31-95f8-2e078ecfb3dd)

Reply:
I changed the names to the states to be more indictive of what stage of the calibration we are at